ensembl_genes/Red.py

```diff
@@ -0,0 +1,264 @@
+""" Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+    Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. """
+
+import eHive
+
+import os
+import filecmp
+import errno
+import subprocess
+
+import sqlalchemy as db
+import sqlalchemy_utils as db_utils
+
+# sqlalchemy requires MySQLdb but MySQLdb doesn't support Python 3.x
+# pymysql can be imported and used instead
+import pymysql
+pymysql.install_as_MySQLdb()
+
+class Red(eHive.BaseRunnable):
```

Comment:
I can see that as a separate project/pull-request but I think we'd need to define better how it would fit in the structure we want for the other modules as well.

Reply:
I think it would be better to start now to define how we want the code to be rather than updating 10 modules in 3 months. Maybe a STS discussion